superchain/superchain.go

```diff
@@ -129,7 +253,26 @@ var Addresses = map[uint64]*AddressList{}
 
 var GenesisSystemConfigs = map[uint64]*GenesisSystemConfig{}
 
+// Implementations represents a global mapping of contract implementations
+// to chain by chain id.
+var Implementations = map[uint64]ContractImplementations{}
+
+// SuperchainSemver represents a global mapping of contract name to desired semver version.
+var SuperchainSemver = ContractVersions{}
```

Comment:
```suggestion
var SuperchainSemver ContractVersions
```
nit: it's a struct, not a map, it's fine to leave it as variable declaration